doc/book/box/limitations.rst

```diff
@@ -49,6 +49,15 @@ Limitations
     such as :ref:`Number of bytes in a tuple <limitations_bytes_in_tuple>`,
     not by the index support.
 
+.. _limitations_fields_in_tuple_multikey_index:
+
+**Number of elements in array fields in a space with a multikey index**
+
+    In a Tarantool space that has :ref:`multikey indexes <box_space-path_multikey>` on array fields,
+    any tuple cannot contain more than ~8,000 elements indexed with that multikey index.
+    This is because every element has 4 bytes of metadata, and the sum of
+    metadata and elements in a tuple cannot exceed 2^16 bytes.
```

Comment:
It isn't sum of elements in a tuple and multikey metadata, it's only sum of all tuple's metadata.

I don't know how to write it correctly, but something like that: `Sum of multikey metadata and other tuple metadata`